features/graph/simple/src/main/java/org/opennms/netmgt/graph/simple/SimpleGraphContainer.java

```diff
@@ -44,6 +44,7 @@
 // TODO MVR probably implement ContainerGraphInfo instead of a property
 // TODO MVR make more type safe
 // TODO MVR why is there a DefaultGraphContainerInfo, but no SimpleGraphContainerInfo... this is weird
+// TODO Patrick make it immutable if possible
```

Comment:
@mvrueden lets discuss if it makes sense to have a container immutable. I assume similar to graph?

Reply:
Yes, we need this as well.